app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -51,6 +51,16 @@ def mfa_enroll_phone_otp_submitted(success:)
       )
     end
 
+    # @param [String] phone_number - the user's phone_number used for multi-factor authentication
```

Comment:
no need for the hyphen
```suggestion
    # @param [String] phone_number the user's phone_number used for multi-factor authentication
```